kubevirt-vmware/pkg/controller/ovirtprovider/client.go

```diff
@@ -276,6 +278,9 @@ func (c *Client) getRaw(sourceVM *ovirtsdk.Vm) (string, error) {
 			return "", err
 		}
 		vnic := profile.(*ovirtsdk.VnicProfile)
+		if vnicID, ok := vnic.Id(); ok {
```

Comment:
Is there a need to keep NetID ?

https://github.com/ManageIQ/manageiq-v2v-conversion_host/pull/51/files#diff-07147f3641032dee71e1db5e7a8999f5R288

Reply:
I think, this was left there to keep it compatible with the old version of UI+vm-import-operator

It can be removed if we switch to VNicID soon

Reply:
I wanted to keep it for now to enable @suomiy to use latest version without need to change the code. We can remove it later when it is not needed any more. Feel free to comment if it is not needed and I will remove it now.

Reply:
It can be removed, but can you please tag an image with old version and keep it around? The image, then can be deleted after we update UI+vm-import-operator.

Reply:
I already uploaded with `latest` tag :cry:

Reply:
doesn't have to be the old one, but just the one which has `NicID` present, so should be okay to use this one

Reply:
OK, I can keep this image and update it when you tell me to.